src/Types/Transaction.purs

```diff
@@ -114,61 +249,9 @@ newtype Address = Address
   }
 
 newtype BaseAddress = BaseAddress
-  { network :: Int, -- u8,
+  { network :: UInt, -- UInt8
```

Comment:
Since we already depend on `arraybuffer-types`, perhaps we could make this a `UInt8` as well

Reply:
https://pursuit.purescript.org/packages/purescript-arraybuffer-types/3.0.1/docs/Data.ArrayBuffer.Types#t:Uint8

in short, UInt8 isn't of kind `Type` it's of kind `ArrayViewType` and `ArrayView` is the only thing of kind `ArrayViewType -> Type`.

we're stuck with UInt